### PR TITLE
New path for tpu end-to-end tests

### PR DESCRIPTION
**Pull request by NinaCai**
# Description

New TPU paths for maxtext end-to-end tests

# Tests

Please describe the tests that you ran on Cloud VM to verify changes.

**Instruction and/or command lines to reproduce your tests:** ...

**List links for your tests (use go/shortn-gen for any internal link):** ...
maxtext_checkpointing: https://540cef12d3da42ce97b48a97401b6c83-dot-us-central1.composer.googleusercontent.com/dags/maxtext_checkpointing/grid

maxtext_convergence: https://540cef12d3da42ce97b48a97401b6c83-dot-us-central1.composer.googleusercontent.com/dags/maxtext_convergence/grid

# Checklist

Before submitting this PR, please make sure (put X in square brackets):
- [x] I have performed a self-review of my code.
- [x] I have necessary comments in my code, particularly in hard-to-understand areas.
- [x] I have run one-shot tests and provided workload links above if applicable. 
- [x] I have made or will make corresponding changes to the doc if needed.

### CODE REVIEW

File: dags/multipod/configs/maxtext_gce_config.py [hunk 1]
```diff
@@ -115,7 +115,7 @@ def get_maxtext_end_to_end_test_config(
 
   test_platform = common.Platform.GCE
   set_up_cmds = common.setup_maxtext(test_mode, test_platform)
-  run_model_cmds = (f"cd /tmp/maxtext && bash end_to_end/{test_script}.sh",)
+  run_model_cmds = (f"cd /tmp/maxtext && bash end_to_end/tpu/{test_script}.sh",)
```
Comment (gobbleturk) [inline dags/multipod/configs/maxtext_gce_config.py:118]: Not sure if this will always be tpu but for now I guess is fine

**Review by gobbleturk — APPROVED** [review]
Thank you for fixing this!

File: dags/multipod/configs/maxtext_gce_config.py (hunk 1 above)
Comment (NinaCai) [reply dags/multipod/configs/maxtext_gce_config.py:118]: This is for TPU only, `job_test_config` below uses TPU.

**Review by jonb377 — APPROVED** [review]
LGTM

**Pull request merged**